sdk/communication/azure-communication-chat/azure/communication/chat/_shared/utils.py

```diff
@@ -8,8 +8,7 @@
 import json
 import calendar
 from typing import cast, Tuple, Optional
-from datetime import datetime
-from azure.core.serialization import TZ_UTC
+from datetime import datetime, timezone
```

Comment:
can you please propagate these changes to all _shared folders of the ACS modalities?

Reply:
Hi @AikoBB , is it okay to do it in a separate PR? it seems like this is more for refactor rather than for our feature?

Reply:
It’s part of our practice to maintain consistency across all shared folders of the ACS SDKs. If a file in one of these shared folders is modified, the same changes should ideally be reflected in the other ACS SDKs as well. It would be great if the changes could be incorporated in this PR. This way, we can avoid forgetting about it later and prevent the folders from becoming unsynchronized. :)

Reply:
resolved in new updates from Anna, closing this